src/transformers/models/trocr/processing_trocr.py

```diff
@@ -61,7 +69,15 @@ def __init__(self, image_processor=None, tokenizer=None, **kwargs):
         self.current_processor = self.image_processor
         self._in_target_context_manager = False
 
-    def __call__(self, *args, **kwargs):
+    def __call__(
+        self,
+        images: ImageInput = None,
+        *,
```

Comment:
Comment for @ArthurZucker: 
Slight breaking change to force users to use keyword arguments in case previously positional args were used. Otherwise, we might either miss some `args` or get it assigned to the wrong keyword parameters and getting not a clear error message.

Reply:
tiny comment: let's add maybe a `#TODO` here to keep in mind to deprecate this pattern in this future :pray: